easybuild/easyconfigs/y/yaff/yaff-1.0.develop.2.14-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1-serial.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "PythonPackage"
+
+name = 'yaff'
+version = '1.0.develop.2.14'
+
+homepage = 'http://molmod.github.io/yaff/'
+description = """Yaff stands for 'Yet another force field'. It is a pythonic force-field code."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['https://github.com/molmod/yaff/releases/download/%(version)s']
+sources = [SOURCE_TAR_GZ]
+
+pyver = '2.7.10'
+
+hdf5ver = '1.8.15-patch1'
+versionsuffix = '-Python-%s-HDF5-%s-serial' %(pyver, hdf5ver)
+
+dependencies = [
+    ('Python', pyver),
+    ('h5py', '2.5.0', versionsuffix),
+    ('molmod', '1.1', '-Python-%s' %(pyver)),
+]
+
+options = {'modulename': name}
+runtest = 'python setup.py build_ext -i; nosetests -v'
```

Comment:
tests are failing because `$DISPLAY` is not set; does this need the same `backend: agg` trick as in the `molmod` easyconfig?